<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MajorOption</class>
 <widget class="QMainWindow" name="MajorOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;
background-color: rgb(255, 208, 142);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_main_liberal_option_title">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>41</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>전공심화</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_main_liberal_ment">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>배달의민족 주아</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string>이미 수강하신 전공심화를 선택하세요. </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right_arrow">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_right_arrow{
background-color: transparent;
border-image: url(:right_arrow3.png);
background: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>781</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔손글씨 펜</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_0_0">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>창의적설계</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_1">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>소프트웨어프로젝트</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_2">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>계산미학</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_3">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>컴퓨터시스템 및 어셈블리언어</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_4">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>객체지향프로그래밍</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_5">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>오토마타와 형식언어</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_6">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>자료구조설계</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_7">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>확률 및 통계</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_8">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>수치해석</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_9">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>오픈소스SW프로젝트</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_10">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>운영체제</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_11">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>컴파일러</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_12">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>휴먼ICT소프트웨어공학</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_0_13">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>공업수학</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_14">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>데이타베이스설계</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_15">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>컴퓨터통신</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_16">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>리눅스시스템응용설계</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_17">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>모바일앱개발</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_18">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>산업체핸즈온프로젝트</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_19">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>웹프로그래밍</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_20">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>캡스톤디자인(1)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_21">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>휴먼ICT인공지능</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_22">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>휴먼인터페이스미디어</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_23">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DB시스템 및 프로그래밍</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_24">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>네트워크 응용 설계</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_25">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>생체인식 및 응용</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox_0_26">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>정보보호이론</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_27">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
</string>
         </property>
         <property name="text">
          <string>캡스톤디자인(2)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_28">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>컨텐츠 기반 무선이동통신</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_29">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>컴퓨터그래픽스</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_30">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>패턴인식</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_31">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>휴먼미디어멀티코어컴퓨터</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_32">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>내장형 시스템설계</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_33">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>휴먼미디어멀티코어컴퓨터</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_34">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>설계패턴</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_35">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>영상처리</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_36">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>정보표준화</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_0_37">
         <property name="font">
          <font>
           <family>배달의민족 주아</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>휴먼인터페이스영상</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_borderline">
    <property name="geometry">
     <rect>
      <x>-19</x>
      <y>-24</y>
      <width>889</width>
      <height>904</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/borderline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="menu_logo.qrc">:/5c06a5c6abde9.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_borderline</zorder>
   <zorder>label_main_liberal_option_title</zorder>
   <zorder>label_main_liberal_ment</zorder>
   <zorder>pushButton_right_arrow</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="menu_logo.qrc"/>
 </resources>
 <connections/>
</ui>
